### PR TITLE
BD-3 feat: add typography

### PR DESCRIPTION
**Pull request by sfgardev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,10 @@
     ]
   },
   "dependencies": {
+    "clsx": "^2.1.1",
     "react": "18.3.1",
-    "react-dom": "18.3.1"
+    "react-dom": "18.3.1",
+    "tailwind-merge": "^2.5.2"
```
Comment (SlavaMryasov) [inline package.json:45]: version needs to be committed

File: package.json [hunk 2]
```diff
@@ -39,8 +39,10 @@
     ]
   },
   "dependencies": {
+    "clsx": "^2.1.1",
```
Comment (SlavaMryasov) [inline package.json:42]: version needs to be committed

**Review by SlavaMryasov — CHANGES_REQUESTED** [review]
versions into package.json needs to be committed + versions from another commits must too be commit

**Comment by sfgardev** [comment]
> versions into package.json needs to be committed + versions from another commits must too be commit

done

**Review by itBanda — APPROVED** [review]

**Review by SlavaMryasov — APPROVED** [review]

**Pull request merged**